app/api/v1/endpoints/tenants.py

```
@@ -159,6 +167,106 @@ async def update_tenant(
     )
 
 
+@router.get("/integrity", response_model=TenantIntegrityStatus)
+async def get_tenant_integrity(
+    tenant_id: Annotated[str, Depends(get_tenant_id)],
+    tenant_repo: Annotated[ITenantRepository, Depends(get_tenant_repo)],
+    history_repo=Depends(get_tenant_integrity_history_repo),
```

Ruff (0.15.5)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)